packages/core/contracts/cross-chain-oracle/chain-adapters/Optimism_ChildMessenger.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.0;
 
 // This should be replaced with a "real" import when Optimism release their new contract versions.
-import "../../external/ovm/OVM_CrossDomainEnabled.sol";
```

Comment:
note this PR also made the following change: rather than using our own `CrossDomainEnabled` file (that was copied directly from Optimism) we now import from their package.